ProjFS.Mac/PrjFSLib.Mac.Managed/Interop/PrjFSLib.cs

```diff
@@ -25,20 +25,17 @@ internal static class PrjFSLib
         [DllImport(PrjFSLibPath, EntryPoint = "PrjFS_WritePlaceholderFile")]
         public static extern Result WritePlaceholderFile(
             string relativePath,
-        
             [MarshalAs(UnmanagedType.LPArray, SizeConst = PlaceholderIdLength)]
             byte[] providerId,
-
             [MarshalAs(UnmanagedType.LPArray, SizeConst = PlaceholderIdLength)]
             byte[] contentId,
-        
             ulong fileSize,
-            UInt16 fileMode);
+            ushort fileMode);

```

Comment:
I'm curious, why did you change this type?

Reply:
@sanoursa I had to update these types after enabling StyleCop for the project

Reply:
:+1: